api/swagger.yaml

```diff
@@ -1577,6 +1577,20 @@ definitions:
           Settable: null
           Value: []
 
+  ObjectVersion:
+    description: |
+      The version number of the swarm related object such as node, service, etc. The version number is
```

Comment:
I'd prefer not to say "swarm related object". Is a secret a swarm related object? Kind of, but there is work underway to eventually let `docker run` use secrets.